### PR TITLE
Add PPA instructions for installing node on Ubuntu

### PR DESCRIPTION
**Pull request by muffinresearch**

### CODE REVIEW

File: docs/install.rst [hunk 1]
```diff
@@ -9,11 +9,23 @@ platform, and there is a version that is written in Python_. Two of
 the services are written in or rely on Python_ the rest are written in
 JavaScript.
 
-If you are on Ubuntu, you should probably build and install Node_ and
-npm_ yourself. The packages that provide binaries of each of these
-names are not the things that you want. apt-get install node gives you
-an amateur packet radio program and npm is provided by a magnetic
-resonence imaging program. Neither of which are what ZenIRCBot needs.
+If you are on Ubuntu, then the quickest and easiert way to get a recent
```
Comment (wraithan) [inline docs/install.rst]: typo, should be easiest

**Comment by wraithan** [comment]
Thanks! If you could fix that typo really quick I'll merge it in :D

**Comment by muffinresearch** [comment]
Good catch - that's fixed!

**Comment by wraithan** [comment]
Thanks!

**Pull request merged**